providers/manage.rb

```diff
@@ -126,6 +126,14 @@ def chef_solo_search_installed?
             mode "0600"
             variables :ssh_keys => u['ssh_keys']
           end
+        else
+          file "#{home_dir}/.ssh/authorized_keys" do
+            action :delete
+          end
+        end
+        # Ensure there are no authorized_keys2 files laying around, conformity.
+        file "#{home_dir}/.ssh/authorized_keys2" do
+          action :delete
```

Comment:
Please don't touch files this cookbook did not create and was never asked to manage.